flink-table/flink-table-runtime/src/test/java/org/apache/flink/table/runtime/functions/table/fullcache/inputformat/FullCacheTestInputFormat.java

```diff
@@ -41,18 +41,22 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-/** TestInputFormat that reads data from (2 + delta) splits which share the same {@code queue}. */
+/**
+ * TestInputFormat that reads data from (initNum + delta) splits which share the same {@code queue}.
```

Comment:
```suggestion
 * TestInputFormat that reads data from ({@link #numSplits} + {@link #deltaNumSplits}) splits which share the same {@code queue}.
```
What is `initNum`? I guess, you were referring to the newly added field `numSplits` analogously to the field `deltaNumSplits`?